src/frontends/onnx/frontend/src/op/com.microsoft/fusedmatmul.cpp

```diff
@@ -0,0 +1,102 @@
+#include "cmath"
+#include "core/operator_set.hpp"
+#include "exceptions.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/transpose.hpp"
+#include "utils/common.hpp"
+
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace com_microsoft {
+namespace opset_1 {
+
+std::vector<size_t> get_transpose_axes(size_t rank, bool transBatch = false) {
+    std::vector<size_t> axes(rank);
+    std::iota(axes.begin(), axes.end(), 0);
+
+    if (transBatch) {
+        if (rank > 2) {
+            std::rotate(axes.begin() + 1, axes.begin() + 2, axes.begin() + rank - 1);
+        }
+    } else {
+        std::swap(axes[rank - 1], axes[rank - 2]);
```

Comment:
please, check size of axes before. it isn't an expected way, but better to prevent out-of-bounds.